### ISSUE TITLE
Recommender: module cloud.google.com/go@latest (v0.46.3) found, but does not contain package cloud.google.com/go/recommender/v1beta1

### ISSUE THREAD

**Issue by jimanvlad** [googleapis/google-cloud-go#1613]
## Client

Recommender apiv1beta1

## Describe Your Environment

Go 1.13 on macOS

## Expected Behavior

Expected the following import statement to work:
`recommender "cloud.google.com/go/recommender/apiv1beta1"`

based on the example here: https://github.com/googleapis/google-cloud-go/blob/master/recommender/apiv1beta1/recommender_client_example_test.go

## Actual Behavior

Go cannot load the module properly:

`build gcp_audit: cannot load cloud.google.com/go/recommender/v1beta1: module cloud.google.com/go@latest (v0.46.3) found, but does not contain package cloud.google.com/go/recommender/v1beta1`

**Comment by jadekler** [comment]
Thanks - we're aware of this and working on fixing it.

**Comment by jadekler** [comment]
This should now be fixed as we've released `v0.47.0`:

```
deklerk at deklerk-macbookpro2 in /tmp/foo
$ go mod init foo
go: creating new go.mod: module foo
deklerk at deklerk-macbookpro2 in /tmp/foo
$ go get -u cloud.google.com/go/...
go: finding github.com/google/pprof latest
go: finding google.golang.org/genproto latest
go: finding golang.org/x/tools latest
go: finding golang.org/x/lint latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/exp latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/sys latest
go: finding golang.org/x/net latest
go: finding golang.org/x/time latest
go: finding github.com/golang/groupcache latest
```

**Comment by jimanvlad** [comment]
Works, thanks!
